tests/foreman/cli/test_host.py

```diff
@@ -117,6 +150,248 @@ def test_positive_create_with_name(self):
                     result['name'],
                 )
 
+    @tier1
+    def test_positive_create_with_org_name(self):
+        """Check if host can be created with organization name
+
+        @id: c08b0dac-9820-4261-bb0b-8a78f5c78a74
+
+        @Assert: Host is created using organization name
+        """
+        new_host = make_fake_host({
+            'content-view-id': self.DEFAULT_CV['id'],
+            'lifecycle-environment-id': self.LIBRARY['id'],
+            'organization': self.new_org['name'],
+        })
+        self.assertEqual(new_host['organization'], self.new_org['name'])
+
+    @run_only_on('sat')
+    @tier1
+    def test_positive_create_with_cv_default(self):
+        """Check if host can be created with content view name
+
+        @id: bb69a70e-17f9-4639-802d-90e6a4520afa
+
+        @Assert: Host is created using content view name
+        """
+        new_host = make_fake_host({
+            'content-view-id': self.DEFAULT_CV['id'],
```

Comment:
Wrong docstring. The test is not about creating using some cv name, but creating using default content view. I'll update the docstring